mobile-widgets/qml/DiveList.qml

```diff
@@ -34,6 +34,18 @@ Kirigami.ScrollablePage {
 			}
 		}
 	}
+//	this will be: onVisibleChanged: {
+	function simulateVisibleChanged() {
+		console.log("---> DiveList changed visibility to " + visible)
```

Comment:
useful during debugging, maybe not something we should keep around.
and yes, I am the absolute worst case offender in keeping them around...
BTW - I have switched to using the `appendTextToLog` function as that at least adds time stamps to the log which makes things easier to read.